src/overlays/actors/ovl_Bg_Haka_Trap/z_bg_haka_trap.c

```diff
@@ -166,7 +166,7 @@ void BgHakaTrap_Init(Actor* thisx, PlayState* play) {
                 this->colliderCylinder.dim.radius = 18;
                 this->colliderCylinder.dim.height = 115;
 
-                this->colliderCylinder.info.toucherFlags = this->colliderCylinder.info.toucherFlags;
```

Comment:
This change doesn't make much sense to me, why do you think this is what it's doing?

Reply:
Because the previous value is `TOUCH_SFX_NORMAL` (`0 << 3`) from the init data
https://github.com/zeldaret/oot/blob/8f1fd58f22518857a1131582af0d41adfde2dad9/src/overlays/actors/ovl_Bg_Haka_Trap/z_bg_haka_trap.c#L57